### ISSUE TITLE
Demand data

### ISSUE THREAD

**Issue by nhniina** [TradeRES/TradeRES-Backbone-demo#11]
Added a ts_influx-time series with demand according to 2050 levels in Nofossil alternative

**Comment by nhniina** [comment]
Fixed BT and BK data

**Comment by nhniina** [comment]
Data from nofossil moved to demand-high

**Comment by SilkeJo** [comment]
rename alternative 2050 -> same for nofossil costs and transmission capacities

**Comment by SilkeJo** [comment]
substract EV demand time series from this time series after initial run

**Comment by nhniina** [comment]
I will check how EVs and other sectors are included in the ENTSO-E data

**Comment by nhniina** [comment]
What ENTSO-E data is this actually?

**Comment by SilkeJo** [comment]
As far as I know, it is based on 2030 and 2050 aggregated demand of Entso-E's global ambition scenario and somehow Ricardo converted them to time series. In doubt, Ni might know more / has access to documentation.

**Comment by nhniina** [comment]
I updated the 2050 consumption data. Originally, the 2050 hourly values seemed to include EV and industrial heat consumption, which were assumed to increase significantly from 2030. Since we add new EV consumption and new industrial hydrogen consumption (due to heat demand) on top of the basic electricity demand, I think it's better to calculate the basic electricity consumption by summing up assumptions on 2050 "conventional demand" and "heat pumps" as well as 2030 "EV" and "industrial heat" consumption (the four categories in the original data). We subtract the heat demand of buildings from the basic electricity consumption in the 2050 scenarios so it makes sense to treat heat pump consumption differently, I think. I also uploaded the new file that calculates the time series to sharepoint.
